### PR TITLE
[merged] Show containers type

### PR DESCRIPTION
**Pull request by giuseppe**
add information about the type of the containers and the images in the "atomic ps" and "atomic images" output

### CODE REVIEW

File: Atomic/ps.py [hunk 1]
```diff
@@ -22,7 +22,7 @@ def ps(self):
             created = dateparse(ret['created']).strftime("%F %H:%M") # pylint: disable=no-member
             all_containers.append({"type" : "systemcontainer", "container" : container,
                                    "image" : image, "command" : command, "created" : created,
-                                   "status" : status})
+                                   "status" : status, "kind" : "runc"})
```
Comment (rhatdan) [inline Atomic/ps.py]: I am not crazy about kind.

machinectl calls this SERVICE

```
MACHINE                          CLASS     SERVICE
ec7dacdbe7859e15c3e4fb2d73226f15 container docker 
```

Comment (rhatdan) [inline Atomic/ps.py]: Perhaps "RUNTIME"?

**Comment by rhatdan** [comment]
I like this patch, but we have to come to some consistency on the naming.  kind, imageKind, type ...

**Comment by giuseppe** [comment]
@rhatdan thanks for the review.  I've addressed the comments in the version here :arrow_up:

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**